src/Pheetup/CoreBundle/Controller/CrudController.php

```diff
@@ -114,9 +115,11 @@ public function viewAction($id)
         $repo = $this->getRepository();
         $view = [];
         $item = $repo->find($id);
-        $view["item"] = (array)$item;;
-        if (!$item->getGroup() instanceof Events) {
-            unset($view["item"][array_keys($view["item"])[6]]);
+        $view["item"] = (array)$item;
+        if (method_exists($item, 'getGroup')) {
+            if ($item->getGroup() instanceof Group) {
+                unset($view["item"][array_keys($view["item"])[6]]);

```

Comment:
Nasıl anlatacağımı bilemedim ama :smile: viewAction'da doctrine ile çekilen veriyi dizi olarak döndüğümüzde, dönen dizinin içindeki groups değişkeni object olarak döndüğü için  [twig](https://github.com/AnkaraPHP/pheetup/blob/master/src/Pheetup/CoreBundle/Resources/views/Crud/view.html.twig)  hataya düşüyordu. Bende groups'u silmekte buldum çareyi :smile: Ne kadar doğrudur bilmiyorum tabi :grin: